contrib/win32/win32compat/inc/sys/un.h

```diff
@@ -1,7 +1,7 @@
-#ifndef COMPAT_UN_H
-#define COMPAT_UN_H 1
+#pragma once
```

Comment:
we should decide on either using #include guard idiom in header files or #pragma once in all the files consistently..